common/src/main/java/com/adaptivescale/rosetta/common/models/enums/OperationTypeEnum.java

```diff
@@ -0,0 +1,10 @@
+package com.adaptivescale.rosetta.common.models.enums;
+
+public enum OperationTypeEnum {
+  PRE_CREATE,
```

Comment:
Everywhere in YAML we use `camelCase`, this part uses all UPPERCASE and _. This part does not make uniform the model.yaml structure.

Any opinion on this?

Reply:
I went through with uppercase  because of it being an Enum Class, but i am open to changing if needed